playbooks/bond.yml

```diff
@@ -0,0 +1,24 @@
+---
+- hosts: all
+  become: True
+  tasks:
+    - nmstate_apply:
+        state:
+          interfaces:
+            - name: bond99
+              type: bond
+              state: up
+              ipv4:
+                address:
+                  - ip: 10.10.10.10
+                    prefix-length: 24
+                enabled: true
+              link-aggregation:
+                mode: balance-rr
+                options:
+                  miimon: '140'
+                slaves:
+                - eth1
+                - eth1.ep
```

Comment:
I think it is not necessary to add the peer of a veth.

Reply:
For a meaningful bond example it would be great to mention two ports, though. eth2 as the name for the second port would be nicer.

Reply:
updated